core/metrics-core-impl/src/test/java/org/hawkular/metrics/core/impl/MetricsServiceITest.java

```diff
@@ -1062,6 +1031,7 @@ private void assertDataRetentionsIndexMatches(String tenantId, MetricType<?> typ
         assertEquals(actual, expected, "The data retentions are wrong");
     }
 
+    @SuppressWarnings("unused")
     private static class VerifyTTLDataAccess extends DelegatingDataAccess {
```

Comment:
There is a lot of code in there that should be useful when we change how TTL works. I will leave in there for now since it is in a test class.